proto/src/chrono.rs

```diff
@@ -1,53 +0,0 @@
-use core::convert::TryInto;
-
-use chrono::{DateTime, Duration, TimeZone, Utc};
```

Comment:
The whole module is removed, rather than being replaced with `time` equivalents, because of the problems detailed below. Conversions from/to `timestamp::Time` can serve as domain-validating, fallible alternatives, until more direct convenience methods are reintroduced here if needed.